domains/Discussions/routes.php

```diff
@@ -39,3 +40,8 @@
     'as' => 'questions.view',
     'uses' => QuestionsViewController::class,
 ]);
+
+Route::get('/questions/{questionId}/answers', [
+    'as' => 'questions.answersList',
```

Comment:
We should stick with the structure we've been using and name this endpoint `questions.answers.list`.